aqt/main.py

```diff
@@ -1107,7 +1107,7 @@ def onCheckMediaDB(self):
             if report:
                 report += "\n\n\n"
             report += _(
-                "In media folder but not used by any cards:")
+                str(len(unused)) + " files found in media folder not used by any cards:")
```

Comment:
The text inside _() must not include dynamic content, as that will prevent translations from being looked up. And if you wish to change the wording, you'll need to switch to using ngettext